src/pages/ContributionsPage/ContributionsPage.tsx

```diff
@@ -133,6 +149,25 @@ const ContributionsPage = () => {
   );
 
   const { control, reset, resetField, setValue } = useForm({ mode: 'all' });
+  const { control: contributionTextControl, handleSubmit } =
+    useForm<contributionTextSchema>({
+      resolver: zodResolver(schema),
+      mode: 'all',
+    });
+  const isAdmin = isUserAdmin(me);
+  const onSubmit: SubmitHandler<contributionTextSchema> = async data => {
+    try {
+      await updateCircle({
+        circle_id: selectedCircle.id,
+        team_sel_text: data.team_sel_text,
+      });
+
+      refetchContributions();
+    } catch (e) {
+      console.warn(e);
```

Comment:
this should show a snackbar error message instead of only showing in the console

Reply:
done